client/actions/requestHandlers/handleLoginrequest.js

```diff
@@ -3,10 +3,12 @@ import jwt from 'jsonwebtoken';
 import setAuthorizationToken from '../../utils/setAuthorization';
 import { SET_CURRENT_USER } from '../types';
 
-export function setCurrentUser(user) {
+
```

Comment:
no jsdoc

Reply:
Also the naming of your actions file is `handleLoginrequest` instead of `handleLoginRequest`